homepage/tests.py

```diff
@@ -1,3 +1,35 @@
-def test_trivial():
+import pytest
+from homepage.models import AppUser, FollowedUserCourses, Course
 
-    assert True
+
+@pytest.mark.django_db
+class TestFollowedUserCourses:
+
+    def test_get_courses_followed_by_appUser(self):

```

Comment:
A comment explaining what the test does will be great here.
Also, maybe it should be divided into several tests? 
Like:
 - test for user with one followed course
 - test for user with multiple followed courses
 - test for user with no followed courses

Reply:
Thanks, good idea
I agree that this method is too long, i will split it into smaller methods

Reply:
regarding the comments - as I've said in the second PR - this is just a request for a short oneliner telling us what you're testing.
For example - the current file has a long test, you've agreed it's good to split it up - and while splitting it into smaller tests will boost readability of the code, a short comment stating what you're testing, above the test method, does not read as clutter to me.
Frankly, if the code had been several tests, I might not have raised this issue if they were testing small enough things, since test names are rather verbose.